README.md

```diff
@@ -36,20 +36,135 @@
 
 ## Usage
 
-1. Clone this repository locally
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Import the NixOS and [home-manager](https://github.com/nix-community/home-manager) modules
+
+<details>
+<summary>With Flakes</summary>
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-22.11";
+    catppuccin.url = "github:catppuccin/nix";
+    home-manager = {
+      url = "github:nix-community/home-manager";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { nixpkgs, catppuccin, home-manager }: let
+    system = "x86_64-linux";
+    pkgs = import nixpkgs {inherit system;};
+  in {
+    # for nixos module home-manager installations
+    nixosConfigurations.host = pkgs.lib.nixosSystem {
+      inherit system;
+      modules = [
+        catppuccin.nixosModules.catppuccin
+        home-manager.nixosModules.home-manager
+        {
+          home-manager.users.user = {
+            modules = [
+              catppuccin.homeManagerModules.catppuccin
+            ];
+          };
+        }
+      ];
+    };
+
+    # for standalone home-manager installations
+    homeConfigurations.user = home-manager.lib.homeManagerConfiguration {
+      inherit pkgs;
+      modules = [
+        catppuccin.homeManagerModules.catppuccin
+      ];
+    };
+  };
+}
+```
+
+</details>
+
+<details>
```

Comment:
This `<details>` tag isn't closed, so the entire rest of the README is hidden in it.